### PR TITLE
[lorax-composer] Add coverage

### PR DESCRIPTION
**Pull request by atodorov**
Initial test & coverage for the lorax-composer branch

### CODE REVIEW

**Comment by atodorov** [comment]
@bcl same changes for lorax-composer branch as in PR #252

File: Makefile [hunk 1]
```diff
@@ -25,6 +25,13 @@ check:
 	@echo "*** Running pylint ***"
 	PYTHONPATH=$(PYTHONPATH):./src/ ./tests/pylint/runpylint.py
 
+test:
+	@echo "*** Running tests ***"
+	PYTHONPATH=$(PYTHONPATH):./src/ $(PYTHON) -m nose -v --with-coverage --cover-erase --cover-branches \
+                        --cover-package=pylorax --cover-package=bin --cover-package=sbin --cover-inclusive
+	coverage report -m
+	[ -f "/usr/bin/coveralls" ] && [ -n "$(COVERALLS_REPO_TOKEN)" ] && coveralls || echo
```
Comment (bcl) [inline Makefile]: Where does coveralls come from? I don't see it in RHEL7 or in EPEL7.
We need to make sure that all of the tests can be run from a RHEL7 system without adding a bunch of 3rd party binary blobs (I think we should be ok using EPEL stuff, eg. pylint, but should limit it to that)

Comment (atodorov) [reply Makefile]: python-coveralls from EPEL

Comment (bcl) [reply Makefile]: Ok, I see python-coverage but not python-coveralls. I initially thought they were the same, but see they aren't so this looks ok.

Comment (atodorov) [reply Makefile]: I see what's happening now. The python-coveralls package is built for EPEL but the last (only) available build has been removed, see:
https://koji.fedoraproject.org/koji/packageinfo?packageID=22032

Still I think if we want to use this service (I want) we can install one of the packages from Fedora or ask the EPEL maintainer to rebuild the package.

Comment (bcl) [reply Makefile]: Yeah, we could also help maintain it if we're going to depend on it.

**Pull request merged**